tests/testAutopas/tests/selectors/tuningStrategy/PredictiveTuningTest.cpp

```diff
@@ -0,0 +1,584 @@
+                                   autopas::DataLayoutOption::soa, autopas::Newton3Option::disabled),
+            predictiveTuning.getCurrentConfiguration());
+  predictiveTuning.addEvidence(20, iteration);
+  ++iteration;
+
+  predictiveTuning.tune();
+  EXPECT_EQ(autopas::Configuration(autopas::ContainerOption::linkedCells, 1., autopas::TraversalOption::sliced,
+                                   autopas::DataLayoutOption::soa, autopas::Newton3Option::disabled),
+            predictiveTuning.getCurrentConfiguration());
+
+  predictiveTuning.reset();
+
+  EXPECT_EQ(autopas::Configuration(autopas::ContainerOption::linkedCells, 1., autopas::TraversalOption::c08,
+                                   autopas::DataLayoutOption::soa, autopas::Newton3Option::disabled),
+            predictiveTuning.getCurrentConfiguration());
+  predictiveTuning.addEvidence(3, iteration);
+  ++iteration;
+
+  predictiveTuning.tune();
+  EXPECT_EQ(autopas::Configuration(autopas::ContainerOption::linkedCells, 1., autopas::TraversalOption::sliced,
+                                   autopas::DataLayoutOption::soa, autopas::Newton3Option::disabled),
+            predictiveTuning.getCurrentConfiguration());
+  predictiveTuning.addEvidence(2, iteration);
+  ++iteration;
+
+  predictiveTuning.tune();
+  EXPECT_EQ(autopas::Configuration(autopas::ContainerOption::linkedCells, 1., autopas::TraversalOption::c01,
+                                   autopas::DataLayoutOption::soa, autopas::Newton3Option::disabled),
+            predictiveTuning.getCurrentConfiguration());
+  predictiveTuning.addEvidence(20, iteration);
+  ++iteration;
+
+  predictiveTuning.tune();
+  EXPECT_EQ(autopas::Configuration(autopas::ContainerOption::linkedCells, 1., autopas::TraversalOption::sliced,
+                                   autopas::DataLayoutOption::soa, autopas::Newton3Option::disabled),
+            predictiveTuning.getCurrentConfiguration());
+
+  predictiveTuning.reset();
+
+  EXPECT_EQ(autopas::Configuration(autopas::ContainerOption::linkedCells, 1., autopas::TraversalOption::c08,
+                                   autopas::DataLayoutOption::soa, autopas::Newton3Option::disabled),
+            predictiveTuning.getCurrentConfiguration());
+}
+
+// Tests the first tuning iteration. There is no prediction and the whole searchSpace should be tested.
+TEST_F(PredictiveTuningTest, testTuneFirstIteration) {
+  unsigned int iteration = 0;
+  autopas::PredictiveTuning predictiveTuning(
+      {autopas::ContainerOption::linkedCells}, {1.},
+      {autopas::TraversalOption::c08, autopas::TraversalOption::c01, autopas::TraversalOption::sliced},
+      {autopas::DataLayoutOption::soa}, {autopas::Newton3Option::disabled}, iteration);
+
+  predictiveTuning.reset();
+
+  EXPECT_EQ(autopas::Configuration(autopas::ContainerOption::linkedCells, 1., autopas::TraversalOption::c08,
+                                   autopas::DataLayoutOption::soa, autopas::Newton3Option::disabled),
```

Comment:
same as above concerning variables for configurations. You could also move them into `PredictiveTuningTest.h` so you only have to define them once.